### ISSUE TITLE
[BUG] Parsing issue with script to query FQDN

### ISSUE THREAD

**Issue by 7MinSec** [ukncsc/lme#13]
**Describe the issue**
In the `update.bat` file there's a `wmic` statement to query the domain's FQDN, and it doesn't seem to parse properly, and therefore is saved as a variable that breaks when used later in the script.

**To Reproduce**
1. Run this command (within the .bat):

````
(wmic computersystem get domain | findstr /v Domain | findstr /r /v "^$") > fqdn.txt
````

2. The fqdn.txt result (for me) is:

````
7min.sec 


````

(There are extra spaces after "7min.sec" and several carriage returns in the fqdn.txt)

**Expected behavior**
A clean `7min.sec` saved in the fqdn.txt

**Screenshots**
See the fqdn.txt, which has extra spaces:

<img width="203" alt="Screen Shot 2019-05-02 at 2 31 51 PM" src="https://user-images.githubusercontent.com/17596787/57101534-14940180-6ce7-11e9-8f67-712374a4a127.png">

**Comment by duncan-ncc** [comment]
This has already been highlighted in #4 for which a fix is currently being tested.

**Comment by duncan-ncc** [comment]
Fixed in #14

**Comment by 7MinSec** [comment]
Thanks!
